src/cityreader/cityreader.py

```diff
@@ -14,16 +21,30 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
 cities = []
 
-def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+with open('cities.csv') as csv_file:
+    csv_reader = csv.reader(csv_file, delimiter=',')
+    line_count = 0
+    for row in csv_reader:
+        if line_count == 0:
+            #print(f'Column names are {", ".join(row)}')
+            line_count += 1
+        else:
+           # print(f'\tCity: {row[0]}, State: {row[1]}, County: {row[2]}, Lat: {row[3]}, Lon: {row[4]}, Population: {row[5]}, Density: {row[6]}, Timezone: {row[7]}, Zips: {row[8]}')
+            line_count += 1
+
+            
+            cities.append(City(row[0], row[3], row[4]))
```

Comment:
The `lat` and `lon` values should be in floats. To pass the test, update the code like this:
cities.append(City(row[0], float(row[3]), float(row[4])))